### ISSUE TITLE
How to change values separator?

### ISSUE THREAD

**Issue by rupesh28** [bostrowski/Pretius-APEX-Enhanced-Lov-Item#22]
Hello,

Is there any setting available after importing the plugin - that will let me change the delimiter for the selected values.

Thanks for your support.

Regards,
Rupesh R

**Comment by bostrowski** [comment]
@rupesh28 
No, there is no option to do it from APEX App Builder. The intention was to be consistent with other native components where the default separator is ":".

But if you need to change it for the whole application you can just modify the pl/sql source and javascript file. At the moment:

- in javascript it is just one variable C_VALUE_SEPARATOR.
- in PL/SQL there are multiple lines where separator is used.

For the future release, I will consider exposing separator as the plugin attribute. 

Btw. What is the reason behind that requirement?

**Comment by rupesh28** [comment]
@bostrowski Thanks for your response. I am trying to use multiples values selected in the list directly in a select query, to populate a grid. I was thinking if I can change the separator then I can directly pass it to the query without having to do any parsing.

**Comment by bostrowski** [comment]
@rupesh28 
the current release doesn't support IG. The next release which is almost finished will support it.
